src/build.js

```diff
@@ -1,6 +1,6 @@
 const _ = require('underscore');
 const path = require('path');
-const log = require('apify-shared/log');
+const log = require('@apify/log').default;
```

Comment:
Why is the `default` needed?

Reply:
Because `require('some-module')` loads whatever is in `some-module`'s `module.exports`, and the default export is, according to the spec, just an export named `"default"`. `import` can deal with this automatically, if you want to import the "whole module", it reads the `default` export from `module.exports`, but `require()` can't.

It worked before because Babel had a hack that hack that if a module had only the default export and no other exports, it would assign `module.exports = theDefaultExport`, so that you could do the straightforward `require()`s without the `.default` part. But now that we switched `apify-shared` to TypeScript and lerna, we don't use Babel anymore, so the `require()`s of default exports have to be done like this.